v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/KafkaToBigQuery.java

```diff
@@ -483,7 +483,7 @@ public static PipelineResult runAvroPipeline(
             .apply(
                 "ReadFromKafka",
                 KafkaTransform.readAvroFromKafka(
-                    bootstrapServers, topicsList, kafkaConfig, avroSchema, null))
+                    bootstrapServers, topicsList, kafkaConfig, avroSchema, null, false))
```

Comment:
Not sure how much this template is used still, but it would seem to me like this pipeline should provide `KafkaReadOptions.getEnableCommitOffsets()` instead of `false`.